skype/skype-ps/skype/Set-CsTeamsAppPermissionPolicy.md

```diff
@@ -138,11 +204,12 @@ Accept wildcard characters: False
 ```
 
 ### -GlobalCatalogAppsType
-Do not use.
+Choose to allow or block the installation of third-party apps.
+
 
 ```yaml
 Type: String
-Parameter Sets: (All)
+Parameter Sets: AllowedAppList,BlockedAppList

```

Comment:
```suggestion
Parameter Sets: (All)
```

I agree with @get-itips. Parameter sets should be (All). AllowedAppList and BlockedAppList should be possible values in the description.